### PR TITLE
New UI + e2e tests

### PR DESCRIPTION
**Pull request by kieran-clifton-civica**
New UI tests and e2e tests for most recent update
Made changes to breaking tests
Removed a couple feature files

### CODE REVIEW

File: src/features/foodType.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        And I open the url "food-type"
+        When I click on the element "foodType.button"
+        And I click on the element "foodType.sixthCheckbox"
+        Then I expect that checkbox "foodType.sixthCheckbox" is checked
+        When I click on the element "foodType.button"
+        And I pause for 1000ms
+        Then I expect the url to not contain "food-type"
+
+         @food_type_page_invalid
+    Scenario: food type page check invalid selection
```
Comment (paul-burnett) [inline src/features/foodType.feature]: Rename this test and also do same test for 'None' option.
Same for all food_type, processing_activities, and business_scale

File: src/features/registrationSummary.feature [hunk 2]
```diff
@@ -35,3 +43,4 @@ Feature: Summary
     And I expect that element "registrationSummary.businessType" contains the text "Fferm da byw"
     And I click on the element "registrationSummary.button"
     Then I expect the url to not contain "registration-summary"
```
Comment (paul-burnett) [inline src/features/registrationSummary.feature:48]: Would be good to check the new fields show in welsh

File: src/features/registrationSummary.feature [hunk 3]
```diff
@@ -9,6 +9,14 @@ Feature: Summary
...
 
+  @SDB-8_happy_path_navigation
+  Scenario: navigate to declaration page
+    Given I open the url "/cleansession"
+    And I go to a special QA page at url "/qa/registration-summary" with injected "registration-summary-partnership-2" data
+    Then I expect the business scale element to contain the text "Don't know"  
+    Then I expect the food type element to contain the text "Don't know"  
+    Then I expect the processing activities element to contain the text "Don't know"
+
```
Comment (paul-burnett) [inline src/features/registrationSummary.feature:19]: Can we have a test where it shows multiple selected items for each of the new fields?
Also check the operator birthdate?

File: src/steps/then.js [hunk 4]
```diff
@@ -161,3 +161,32 @@ Then(
...
+    const elementText = await foodTypeElement.getText();
+    expect(elementText).toContain(text);
+  }
+);
+
+Then(
+  "I expect the processing activities element to contain the text {string}",
+  async function (text) {
+    const processingActivitiesElement = await $(
+      "dt=Processing Activities"
```
Comment (paul-burnett) [inline src/steps/then.js]: Frontend is being corrected to lower case 'a' so this will also need updating

**Review by paul-burnett — APPROVED** [review]
